src/pages/MoviesPage.vue

```diff
@@ -40,6 +40,11 @@ import SelectInput from "../components/SelectInput.vue";
 import ActionBar from "../components/ActionBar.vue";
 
 export default {
+  name: "MoviesPage",
+  metaInfo: {
+    title: "Movies",
+    titleTemplate: "%s | Monte Cinema",
```

Comment:
Same `titleTemplate: "%s | Monte Cinema"` is in top App file. 
If you don't want to overwrite it, it will be applied either way